src/Columns/Taxonomy/AbstractTaxonomyColumns.php

```diff
@@ -46,13 +46,12 @@ abstract public function addColumnName(array $columns): array;
 	/**
 	 * Render the taxonomy column content in the custom taxonomy column
 	 *
-	 * @param string $_string Blank string.
 	 * @param string $columnName Name of the column.
 	 * @param int    $termId Term ID.
 	 *
 	 * @return string The content to display in the custom column.
 	 */
-	abstract public function renderColumnContent(string $_string, string $columnName, int $termId): string;
+	abstract public function renderColumnContent(string $columnName, int $termId): string;
```

Comment:
This is not correct: https://developer.wordpress.org/reference/hooks/manage_this-screen-taxonomy_custom_column/

The first argument of the filter is the thing that is filtered out, and the first argument is the custom column output

https://core.trac.wordpress.org/browser/tags/5.9/src/wp-admin/includes/class-wp-terms-list-table.php#L638

So the better naming would be 
```suggestion
	abstract public function renderColumnContent(string $columnOutput, string $columnName, int $termId): string;
```